WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectView.swift

```diff
@@ -98,7 +98,7 @@ final class CardReaderSettingsConnectView: NSObject {
 
     private func configureButton(cell: ButtonTableViewCell) {
         let buttonTitle = NSLocalizedString(
-            "Connect card reader",
+            "Connect Card Reader",
```

Comment:
Not a blocker, but we usually declare strings as static properties a private, inner enum, I think the conventions to name it Localization. There are plenty of examples around.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3950/commits/26b6425e46cb4523f3e2f99f1b078ab941be81fc